testng-core/src/test/java/test/configurationfailurepolicy/issue2731/ConfigFailTestSample.java

```diff
@@ -0,0 +1,33 @@
+package test.configurationfailurepolicy.issue2731;
+
+import org.testng.Assert;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class ConfigFailTestSample {
```

Comment:
This is where I have a concern.

The documentation says this `Whether TestNG should continue to execute the remaining tests in the suite or skip them if an @Before* method fails. Default behavior is skip.`

But here we are trying to move forward with trying to run the next lower level configuration even when the earlier higher order one failed.

Reply:
> Whether TestNG should continue to execute the remaining tests in the suite or skip them if an @Before* method fails. Default behavior is skip.

In my opinion, the default behavior is skip because of that the default config policy is set to `FailurePolicy DEFAULT_CONFIG_FAILURE_POLICY = FailurePolicy.SKIP;` , I'm not sure the description for `Continue`, will look up in document.